troposphere/static/js/components/modals/instance/launch/components/BasicInfoForm.react.js

```diff
@@ -5,22 +5,28 @@ import SelectMenu from 'components/common/ui/SelectMenu.react';
 
 export default React.createClass({
 
+    // TODO: Its not clear what the interface for this method is, what props
+    // do i need to pass? One way to resolve this is to fill out prop types,
+    // or to make aliases to every prop at the top of the func, like 
+    //  var im = this.props.imageVersion
    render: function () {
         if (!this.props.imageVersion || !this.props.imageVersionList) {
             return (<div className="loading"/>);
         }
 
-        let name = function(item) { return item.get('name') };
```

Comment:
Is it because it is unclear that it is for a callback the child will use? Is that why you inline it, so one can see that it is passed to the child for use?